static/js/app.js

```diff
@@ -132,6 +132,9 @@ function deleteAliasConfirmation(submitEvent) {
     addressEl.textContent = aliasToDelete;
   });
 
+  const aliasToDeleteLabel = document.querySelector(".modal-message strong");
```

Comment:
Update the query selector to the new `js-` class you added above. 

Also, please add a comment above explaining what is happening.

Reply:
Okay – Leave it as is: `document.querySelector(".modal-message strong");`